Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -16,23 +16,23 @@ Alpha signals that we are ready for early feedback, that this is not the final v
+Software systems cause emissions through the hardware that they operate on, both through the energy that the physical hardware consumes and the emissions associated with manufacturing the hardware. The Software Carbon Intensity (SCI) Specification defines a methodology for calculating the rate of carbon emissions for a software system. The purpose is to help users and developers make informed choices about which tools, approaches, architectures, and services they use in the future. It is a score rather than a total; lower numbers are better than higher numbers, and reaching 0 is impossible. This specification is focused on helping users and developers understand how to improve software to reduce or avoid the creation of emissions.
 
-Reducing an SCI score is only possible through the elimination of emissions through changes to a software system to use less physical hardware, less energy, or consume lower-carbon energy sources. Neutralization or avoidance offsets do not reduce an SCI score ([see exclusions section](#exclusions)). This makes the SCI an ideal strategy organizations can adopt to meet climate targets focussed on eliminating emissions, such as a [Net-zero target](https://sciencebasedtargets.org/net-zero).
+Reducing an SCI score is only possible through the elimination of emissions. That can be achieved by changes to a software system to use less physical hardware, less energy, or consume lower-carbon energy sources. Neutralization or avoidance offsets do not reduce an SCI score ([see exclusions section](#exclusions)). This makes the SCI an ideal strategy that organizations can adopt to meet climate targets focussed on eliminating emissions, such as a [Net-zero target](https://sciencebasedtargets.org/net-zero).
 
 The SCI is for everyone. It is possible to calculate an SCI score for any software application, from a large distributed cloud system to a small monolithic open source library, any on-premise application or even a serverless function. The product or service may be running in any environment, whether a personal computer, private data center or a hyperscale cloud.
 
-As software practitioners, we all have a significant role to play in collectively reducing the SCI Score during the design, development and delivery of software applications. The following list provides some strategies that can be used to reduce the SCI score across software roles. 
+As software engineers, we all have a significant role to play in collectively reducing the SCI Score during the design, development and delivery of software applications. The following list provides some strategies that can be used to reduce the SCI score across software roles. 
```

Comment:
Engineers has a very specific connotation in some parts of the world and hence it might artificially restrict the audience of the SCI. See here for a discussion: https://news.ycombinator.com/item?id=2247021 

I would suggest to keep the wording as is for this.